region/brinstar/pink/Pink Brinstar Hopper Room.json

```diff
@@ -1504,6 +1504,308 @@
       "wallJumpAvoid": true,
       "note": "Shinesparking horizontally into the wall will trigger the elevator as Samus's echos hit it."
     },
+    {
+      "link": [2, 3],
+      "name": "Side Platform Cross Room Jump",
+      "entranceCondition": {
+        "comeInWithSidePlatform": {
+          "platforms": [
+            {
+              "minHeight": 1,
+              "maxHeight": 1,
+              "obstructions": [[3, 0]],
+              "minTiles": 36.3703,
+              "speedBooster": true,
+              "environment": "water",
+              "requires": [
+                "canTrickyDashJump",
+                "canInsaneJump",
+                "canGravityJump",
+                "canMomentumConservingMorph"
+              ],
+              "note": ["This applies to Botwoon Energy Tank Room"],
+              "detailNote": [
+                "This needs extra run speed of $6.0 or $6.1.",
+                "It is easier to do by timing a pause to hit just after jumping and aiming down,",
+                "then unequipping Gravity and holding down during the unpause black screen to buffer the morph."
+              ]
+            }
+          ]
+        }
+      },
+      "requires": [],
+      "clearsObstacles": ["B"],
+      "wallJumpAvoid": true
+    },
+    {
+      "link": [2, 3],
+      "name": "Side Platform Cross Room Jump (Hi-Jump)",
+      "entranceCondition": {
+        "comeInWithSidePlatform": {
+          "platforms": [
+            {
+              "minHeight": 1,
+              "maxHeight": 1,
+              "obstructions": [[1, 0]],
+              "minTiles": 16,
+              "speedBooster": true,
+              "requires": [
+                "canTrickyDashJump"
+              ],
+              "note": ["This applies to Lava Dive Room"],
+              "detailNote": [
+                "This needs extra run speed between $2.C and $2.F."
+              ]
+            },
+            {
+              "minHeight": 2,
+              "maxHeight": 2,
+              "obstructions": [[1, 0]],
+              "minTiles": 27.4375,
+              "speedBooster": true,
+              "note": ["This applies to Dust Torizo Room and Noob Bridge"]
+            },
+            {
+              "minHeight": 3,
+              "maxHeight": 3,
+              "obstructions": [[1, 0]],
+              "minTiles": 22,
+              "speedBooster": true,
+              "requires": [
+                "canTrickyDashJump"
+              ],
+              "note": ["This applies to Double Chamber"],
+              "detailNote": [
+                "Stand between about 1 and 7 tiles away from the runway end, to start running.",
```

Comment:
`1-7 tiles` is enough to not need trickyDashJump, but there are a lot of small things to move this towards expert.  Most bigly is that the full runway does not work and looking up where to stand is not the ideal for very hard.